src/main/java/com/ldtteam/structurize/blocks/AbstractBlockStructurizeWall.java

```diff
@@ -0,0 +1,43 @@
+package com.ldtteam.structurize.blocks;
+
+import com.ldtteam.structurize.api.util.constant.Suppression;
+import com.ldtteam.structurize.blocks.interfaces.IBlockStructurize;
+import net.minecraft.block.Block;
+import net.minecraft.block.WallBlock;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+import net.minecraftforge.registries.IForgeRegistry;
+
+/**
+ * The abstract class for structurize-added walls.
+ * Currently only used by brown brick, beige brick,
+ * and cream brick walls.
+ */
+
+public abstract class AbstractBlockStructurizeWall<B extends AbstractBlockStructurizeWall<B>> extends WallBlock implements IBlockStructurize<B>
+        /**
+         * Extend the vanilla WallBlock

```

Comment:
this javadoc comment is "in the middle of nowwhere"

There should probably be one above the constructor, and with an @parameter